README.md

```diff
@@ -137,8 +140,12 @@ Put your audio file in mp3 format in `/audio` and replace `audio` with the file
 Please note that you <b>can</b> have both audio and alert box in your tile as well.
 
 <br>
-And replace `title` and `text` with whatever title or text you want to display.
-If you want to only have a title and not any text, keep `text` empty. Like this : `""`.
+
+If you want to add a background music, add your music to `audio/background/` in mp3 format
```

Comment:
I think, this can go into a Music Section.

Reply:
Done!